File: src/main/java/entity/Message.java
package entity;

import io.quarkus.hibernate.orm.panache.PanacheEntity;

import javax.persistence.*;

@Entity
@Table(name = "message")
//@SecondaryTable(name = "user")
public class Message extends PanacheEntity {

    @Column(name = "message_title")
    public String title;

    @Column(name = "message_text")
    public String text;

    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "user_id")
    public User user;

//    @Column(name = "email", table = "user")
//    public String email;

    public Message() {

    }

    public Message(String title, String text, User user) {
        this.title = title;
        this.text = text;
        this.user = user;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getText() {
        return text;
    }

    public void setText(String text) {
        this.text = text;
    }

    public User getUser() {
        return user;
    }

    public void setUser(User user) {
        this.user = user;
    }
}

File: webapp/src/app/pages/Message.ts
export class Message {


  private _id :number;
  private _title: string;
  private _text: string;
  private _user: object;


  constructor(id: number, title: string, text: string, user: object) {
    this._id = id;
    this._title = title;
    this._text = text;
    this._user = user;
  }

  get id(): number {
    return this._id;
  }

  set id(value: number) {
    this._id = value;
  }

  get title(): string {
    return this._title;
  }

  set title(value: string) {
    this._title = value;
  }

  get text(): string {
    return this._text;
  }

  set text(value: string) {
    this._text = value;
  }

  get user(): any {
    return this._user;
  }

  set user(value: any) {
    this._user = value;
  }

}
